package it.pagopa.selfcare.pagopa.backoffice.connector.exception;

public class ResourceAlreadyExistsException extends RuntimeException{

    public ResourceAlreadyExistsException(String msg, Throwable cause) {
        super(msg, cause);
    }

}






